packages/friends/yml/en/john-pemberton.yml

```diff
@@ -1,15 +1,35 @@
 name: John Pemberton
 slug: john-pemberton
 gender: male
-description: TODO
+description:
+  The ​​simplicity, meekness and diffidence​ of John Pemberton (1727-1795)​, united with
+  an ardent love for his fellow creatures​ and a desire for their everlasting welfare​,
+  made him a eminent and much-beloved minister in the Society of Friends. ​​​An abiding
+  sense of the Divine presence seemed to cover his spirit wherever he traveled; and though
+  his journal speaks much of his weakness and struggles, and little of his gifts and great
+  usefulness in the church of God, yet the reader will be amply repaid for tracing the
+  footsteps of a man much weaned from the spirit and​ ways of the world, who eagerly
+  endeavoured to perform his day’s work in the day time, looking not for the acceptance of
+  man, but for the testimony of a clean conscience in the faifthful service of his Lord
+  and Master.
```

Comment:
Do you think we could work this quote from Rebecca Jones into the blurb somehow? It comes to mind often when I think of John Pemberton:

> George Dillwyn has sometimes made a comparison between some of us and John Pemberton, that we are as fishing with a crooked pin and thread, while John is casting his net into the sea.

Reply:
I tried. Let me know what you think